### PR TITLE
Add note explaining how to add another node - Closes #439

### LINKED ISSUE #439: [doc] 6.5.3.1  Deploy Ceph Monitors and Ceph Managers
### Type of report
   - [ ] bug
   - [X] feature request

### Source Document
6.5.3.1  Deploy Ceph Monitors and Ceph Managers:

https://susedoc.github.io/doc-ses/master/html/ses-deployment/deploy-cephadm.html#deploy-cephadm-day2-service-mon

### Summary

Would be great to know if I need to add another one if it is enough to add it into the yaml. That is not mentioned here

https://susedoc.github.io/doc-ses/master/html/ses-deployment/deploy-cephadm.html#deploy-cephadm-day2-service-mon
susedoc.github.io

the solution is to change

```yaml
service_type: mon
placement:
  hosts:
  - host1
  - host2
```

to

```yaml
service_type: mon
placement:
  hosts:
  - host1
  - host2
  - host3
```

### Actual Results

`ceph orch apply mon host3` actually deployed a MON on one node but removed some others

### PR DESCRIPTION
**Pull request by asettle**

### CODE REVIEW

**Comment by dthpulse** [comment]
looks good to me
thanks

File: xml/deploy_cephadm.xml [hunk 1]
```diff
@@ -1933,6 +1933,21 @@ placement:
   - ses-min2
   - ses-min3
 </screen>
+     <note>
+       <para>
+         If you need add another node, append the host name to the same yaml list.
```
Comment (lproven) [inline xml/deploy_cephadm.xml:1938]:
```suggestion
         If you need add another node, append the host name to the same YAML list.
```

**Comment by lproven** [comment]
@dthpulse -- I think @asettle is on vacation now so if you are happy with my only proposed change ("yaml" → "YAML") then I will make the change and merge/close this for her.

**Comment by asettle** [comment]
@lproven please reapprove and merge. Was there a reason this was delayed for 11 days? Is there something I'm missing?

**Pull request merged**